CPP_STYLE_GUIDE.md

```diff
@@ -192,40 +192,40 @@ class FancyContainer {
 
 ### Use `nullptr` instead of `NULL` or `0`
 
-What it says in the title.
+Further reading in the [C++ Core Guidelines][ES.47].
 
 ### Ownership and Smart Pointers
 
-"Smart" pointers are classes that act like pointers, e.g.
-by overloading the `*` and `->` operators. Some smart pointer types can be
-used to automate ownership bookkeeping, to ensure these responsibilities are
-met. `std::unique_ptr` is a smart pointer type introduced in C++11, which
-expresses exclusive ownership of a dynamically allocated object; the object
-is deleted when the `std::unique_ptr` goes out of scope. It cannot be
-copied, but can be moved to represent ownership transfer.
-`std::shared_ptr` is a smart pointer type that expresses shared ownership of a
-dynamically allocated object. `std::shared_ptr`s can be copied; ownership
-of the object is shared among all copies, and the object
-is deleted when the last `std::shared_ptr` is destroyed.
-
-Prefer to use `std::unique_ptr` to make ownership
-transfer explicit. For example:
+[R.20]: Use `std::unique_ptr` or `std::shared_ptr` to represent ownership<br/>
+[R.21]: Prefer `unique_ptr` over `shared_ptr` unless you need to share ownership
```

Comment:
If you want a list of things, you can juse start lines with `*` instead of manually inserting line breaks

Reply:
ack x 3